examples/sample_manifest_extended.json

```diff
@@ -0,0 +1,153 @@
+        "role": ["nrt"]
+        },
+        {
+        "name": "Beatrice Narrator",
+        "scheme": "https://www.loc.gov/marc/relators/",
+        "role": ["nrt"]
+        }
+    ],
+
+    "schema:hasPart": [
+        {
+            "@type": "http://schema.org/VisualArtwork",
+            "schema:url": "cover.jpg",
+            "schema:license": "https://creativecommons.org/publicdomain/zero/1.0/"
+        },
+        {
+            "@type": "http://schema.org/Book",
+            "schema:url": "this_is_for_copyright_on_text_copy",
+            "schema:license": "https://creativecommons.org/publicdomain/zero/1.0/"
+        },
+        {
+            "@type": "http://bib.schema.org/Audiobook",
+            "schema:url": "this_is_for_copyright_on_performance",
+            "schema:license": "https://creativecommons.org/publicdomain/zero/1.0/"
+        }
+    ]
+
+    "schema:license": "https://creativecommons.org/publicdomain/zero/1.0/"
+  },
+
+  "links": [
+    {"rel": "self", "href": "https://github.com/HadrienGardeur/audiobook-manifest/blob/master/examples/flatland.audiobook-manifest", "type": "application/audiobook+json"},
+
+    {"rel": "alternate", "href": "https://hadriengardeur.github.io/webpub-manifest/examples/MobyDick/manifest.json", "type": "application/webpub+json"},
+
+    {"rel": "license", "href": "http://example.org/license/35d9b2d6", "type": "application/vnd.readium.lcp.license.v1.0+json"},
+    {"rel": "register", "href": "http://example.org/license/35d9b2d6/register{?id,name}", "type": "application/vnd.readium.license.status.v1.0+json", "templated": true},
+    {"rel": "return", "href": "http://example.org/license/35d9b2d6/return{?id,name}", "type": "application/vnd.readium.lcp.license.v1.0+json", "templated": true},
+    {"rel": "renew", "href": "http://example.org/license/35d9b2d6/renew", "type": "text/html"},
+    {"rel": "renew", "href": "http://example.org/license/35d9b2d6/renew{?end,id,name}", "type": "application/vnd.readium.lcp.license.v1.0+json", "templated": true},
+     
+    
+    {"rel": "cover", "href": "http://ia800704.us.archive.org/9/items/LibrivoxCdCoverArt12/Flatland_1109.jpg", "type": "image/jpeg", "height": 300, "width": 300},
+    {"rel": "alternate", "href": "https://archive.org/download/flatland_rg_librivox/Flatland_64kb_librivox.m4b", "type": "audio/x-m4b", "bitrate": 64},
+    {"rel": "alternate", "href": "http://www.archive.org/download/flatland_rg_librivox/flatland_rg_librivox_64kb_mp3.zip", "type": "application/zip"},
+    {"rel": "alternate", "href": "http://www.archive.org/download/flatland_rg_librivox/flatland_rg_librivox_archive.torrent", "type": "application/x-bittorrent"}
+  ],
+
+  "spine": [
+    {
+        "href": "http://www.archive.org/download/flatland_rg_librivox/flatland_1_abbott.mp3", 
+        "type": "audio/mpeg", 
+        "bitrate": 128, 
+        "duration": 1371, 
+        "title": "Part 1, Sections 1 - 3",
+        "schema:hasPart": [
```

Comment:
You can't do that with our current model. If the goal is to provide the narrator for a specific resource, this should be in the metadata section above as:
```
"schema:hasPart": [
  {
    "schema:url": "http://www.archive.org/download/flatland_rg_librivox/flatland_1_abbott.mp3",
    "narrator": "Abe Narrator"
  }
]
```